### DISCUSSION TITLE
Installation worked but it is not possible to open Events Details

### DISCUSSION THREAD

**Discussion by jessidevil** [craigk5n/webcalendar#390] [Q&A]
I have installed the webcalendar-master on my Server:

|SYSTEM INFORMATION&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| |
|---|---|
|  OS type and version |     Ubuntu Linux 22.04.3     |
|  Kernel and CPU  |  Linux 5.15.0-78-generic auf x86_64  |
|  Webmin-Version  |  2.101  |
|  Usermin-Version  |  2.001  |
|  Virtualmin Version  |  7.7.  |
|  Authentic theme version  |  21.04  |
|  Apache Version  |  2.4.52  |
|  PHP Version  |  8.1.2  |
|  MariaDB Version  |  10.6.12  |

All looks good, you can see Events in the Overview Month, Week, Daily,... but if you click it you will only get a Error 500 from the Server. Any Ideas?

Thanks in Advance :)

**Comment by craigk5n** [comment]
Check your apache error log, normally `/var/log/apache2/error.log`.

**Reply by jessidevil** [reply]
Nothing in the Apache Error Log but i found something in the php_log

[12-Aug-2023 11:38:15 America/Toronto] PHP Fatal error:  Uncaught Error: Class "DocList" not found in /home/mg/public_html/includes/classes/AttachmentList.php:15
Stack trace:
#0 /home/mg/public_html/view_entry.php(18): require_once()
#1 {main}
  thrown in /home/mg/public_html/includes/classes/AttachmentList.php on line 15

**Comment by AxelDuerrbaum** [comment]
Same for me: editing any  entries as owner or admin give Error 500

/apache2/error.log:

[Tue Aug 15 15:54:54.354224 2023] [php7:error] [pid 598700] [client x.x.x.x:52396] PHP Fatal error:  Uncaught Error: Class 'DocList' not found in /opt/WWW/webcalendar/includes/classes/AttachmentList.php:15\nStack trace:\n#0 /opt/WWW/webcalendar/view_entry.php(18): require_once()\n#1 {main}\n  thrown in /opt/WWW/webcalendar/includes/classes/AttachmentList.php on line 15, referer: https://x.x.x.x/webcalendar/week.php
